rask-engine/src/boxes.rs

```diff
@@ -47,16 +48,8 @@ impl ops::SubAssign<Vec2> for AABox {
     }
 }
 
-impl PartialEq for AABox {
-    fn eq(&self, other: &Self) -> bool {
-        self.pos == other.pos && self.size == other.size
-    }
-}
-
-impl Eq for AABox {}
-
 /// A rotated box.
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
```

Comment:
Deriving `Eq`/`PartialEq` does also mean floating point comparison without an epsilon. Why is `Eq` and `PartialEq` needed here?

Reply:
Same at `AABox`

Reply:
Please review engine code on the engine pr